OCR/glm_ocr_pro/README.md

```
@@ -78,6 +103,10 @@ GLM-OCR Pro includes three "Performance Profiles" to help you balance speed and
 
 ---
 
-## 📝 License
+## License
 
 This project is intended for personal and professional use. Ensure compliance with the GLM-OCR and Ollama licensing terms.
+
+---
+
+[Back to top](#glm-ocr-pro)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)